text/0000-ember-new-lang.md

```diff
@@ -0,0 +1,57 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- Relevant Team(s): (fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies)
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Ember New Lang
+
+## Summary
+
+Introduce `--lang` flag as an option for `ember new` within Ember-CLI. This will set the `lang` attribute on the html element. A `lang` attribute defines the language of an element or a document.
+
+## Motivation
+
+This will in part address item 4 within issue https://github.com/emberjs/rfcs/issues/595
```

Comment:
We need to find a concise way to say here that this RFC does *not* intend to assume a default language, rather its aim is to lay the groundwork for a future interactive solution that prompts every ember user to start out of the gate with a language specified.

Reply:
Pushed up e66ba23, let me know what you think and feel free to make any adjustments directly you wish.

Reply:
(hah, typo fix in 61d6e3d)